lib/index.js

```diff
@@ -205,21 +205,21 @@ exports.compile = function(str, options){
   var parsed = parse(str, options);
   if (options.compileDebug !== false) {
     fn = [
-        'var jade_debug = [ new jade.DebugItem( 1, ' + filename + ' ) ];'
+        'var pug_debug = [ new pug.DebugItem( 1, ' + filename + ' ) ];'
       , 'try {'
       , parsed.body
       , '} catch (err) {'
-      , '  jade.rethrow(err, jade_debug[0].filename, jade_debug[0].lineno' + (options.compileDebug === true ? ',' + utils.stringify(str) : '') + ');'
+      , '  pug.rethrow(err, pug_debug[0].filename, pug_debug[0].lineno' + (options.compileDebug === true ? ',' + utils.stringify(str) : '') + ');'
       , '}'
     ].join('\n');
   } else {
     fn = parsed.body;
   }
-  fn = new Function('locals, jade', fn)
+  fn = new Function('locals, pug', fn)
```

Comment:
I'm a bit worried that people are using internal symbols like `jade.escape` and `jade_mixins` themselves. Changing the parameter name might break these templates. @ForbesLindesay, what's your opinion on this?